_posts/2019-05-28-community-survey.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 2019 Pulp Communiy Survey
+author: David Davis
+tags:
```

Comment:
this doesn't have to be here, but iirc there is a pulp3 tag (maybe in previous posts)